chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -828,6 +829,8 @@ sync.localDataWillBeCombined		= If you continue, local Zotero data will be combi
 sync.localGroupsWillBeRemoved1		= Local groups, including any with changed items, will also be removed from this computer.
 sync.avoidCombiningData				= To avoid combining data, revert to the '%S' account or use the Reset options in the Sync pane of the Zotero preferences.
 sync.unlinkWarning					= Are you sure you want to unlink this account?\n\n%S will no longer sync your data, but your data will remain locally.
+sync.warning.emptyLibrary			= Your are about to sync your account to an empty %1$S database. This could happen if you removed your previous %1$S database or if the location of your data directory changed.
```

Comment:
Says *your account* instead of *the '%1$S' account*, because with empty database there is no reliable way to get the account. We could use prefs, but it does not have to be populated.